<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationView</class>
 <widget class="QWidget" name="OperationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="wdg_precise" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lab_precise_name">
        <property name="text">
         <string>选取时间段：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dte_start">
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_precise_mid">
        <property name="text">
         <string>至</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dte_end">
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdg_peek" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lab_peek_name">
        <property name="text">
         <string>输入限值：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_peek"/>
      </item>
      <item>
       <widget class="QLabel" name="lab_peek_unit">
        <property name="text">
         <string>dB</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdg_modify" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lab_modify_name">
        <property name="text">
         <string>将所有数据相对调整</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_modify"/>
      </item>
      <item>
       <widget class="QLabel" name="lab_modify_unit">
        <property name="text">
         <string>dB</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdg_button" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_confirm">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
